### DISCUSSION TITLE
Кратко про преподавателей

### DISCUSSION THREAD

**Discussion by guzeradimazz** [asiliuk/BsuirScheduleApp#42] [Ideas]
Добавить к каждому преподавателю его "характеристику". Как он ведёт, фичи, стоит-ли пропускать, способ получения "автомата" по предмету. И многое другое...

Полезно будет для первачей, чтобы они смогли как можно скорее освоиться и привыкнуть.

Хочу в топ😎🤩

**Comment by asiliuk** [comment]
Это будет тяжело сделать, т.к. потребует какой-то бэк-енд с базой и возможностью туда писать всем плюс модерация и все такое
 В свое время мы пользовались http://bsuir-helper.ru/lectors, не знаю на сколько он сейчас актуален
